Shipr-Containerization/shipr-inventory/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+COPY requirements.txt /app/requirements.txt
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app

```

Comment:
I think using `WORKDIR` directive and then explicitly mentioning the same directory serves no purpose. Any thoughts on this?

Reply:
Using of . is more feasible